README.md

```diff
@@ -53,7 +53,6 @@ Options:
   -disable-keepalive    Disable keep-alive, prevents re-use of TCP
                         connections between different HTTP requests.
   -cpus                 Number of used cpu cores.
-                        (default for current machine is 8 cores)
   -host                 HTTP Host header.
   -http-trace         	Enable http trace detailed info on various events during request (Experimental)
```

Comment:
could you also replace this with -more. -http-trace is outdated, it is now called -more.